.sbtopts

```diff
@@ -0,0 +1,3 @@
+-J-Xmx2G
+-J-Xms2G
```

Comment:
Did you try running 
`;clean;reload;update;compile;test` with  this settings? I think it will overflow.

Reply:
It seemed to fix things for me after the overflows I had, but I'll try your scenario as well :) Did it overflow for you? What are your memory settings?

Reply:
I usually use `5G` just because I can ;) but I know people had problem with tests at 2 gigs.

Reply:
For the others reading this: the change only affects default settings, so your custom ones should still be applied automatically.

For those without custom settings, I've tested the build several times and had no OOM errors since applying those settings. Will update them if I spot anything OFC :)